cmd/task/task.go

```diff
@@ -81,6 +83,7 @@ func main() {
 	pflag.BoolVarP(&list, "list", "l", false, "Lists tasks with description of current Taskfile.")
 	pflag.BoolVarP(&listAll, "list-all", "a", false, "Lists tasks with or without a description.")
 	pflag.BoolVarP(&listJson, "json", "j", false, "Formats task list as JSON.")
+	pflag.StringVar(&taskSort, "sort", "", "Changes the order of the tasks when listed.")
```

Comment:
We need to update the API documentation page to include this flag.

Reply:
Unless I'm missing something, this is already updated?

Reply:
Oh, I missed this change when reviewing yesterday.

Ready to merge, then. 👍